src/bootstrap-table.js

```diff
@@ -814,7 +818,7 @@ class BootstrapTable {
         return
       }
 
-      if ($(currentTarget).hasClass('search-input')) {
+      if (currentTarget === Utils.getSearchInput(this)[0] || $(currentTarget).hasClass('search-input')) {
```

Comment:
Can we compare with id or class? instead of object comparison

Reply:
I also thought about that, but then have to parse the dom selector and check the correct proberty which could quite complex.